src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -265,6 +265,7 @@ static string GenerateTestTimedOutMessage(TimeSpan maxTime)
             {
                 throw new Exception("Starting endpoints took longer than 2 minutes");
             }
+            await completedTask.ConfigureAwait(false);
```

Comment:
What is the intention of this change?

`Task.WhenAll` returns `Task<Task>` so the result of the task created by `Task.WhenAll` is the first already completed Task. 

Is your intention to rethrow any exceptions that might have happened in the `whenAll`?